src/echarts.js

```diff
@@ -1142,8 +1142,22 @@ echartsProto.resize = function (opts) {
     flushPendingActions.call(this, silent);
 
     triggerUpdatedEvent.call(this, silent);
+
+    hideTooltip.call(this);
 };
 
+/**
+ * when alwaysShowContent is true change or rotation window size and restore will hide tooltip
+ */
+function hideTooltip() {
+    const tooltips = this._componentsViews.filter(item => item.__model.mainType === 'tooltip');
```

Comment:
Also, any property started by '_' are private, so we shouldn't directly manipulate it

Reply:
Currently I have no other way to hide tooltip in resie.

Reply:
> Also, any property started by '_' are private, so we shouldn't directly manipulate it

Currently I have no other way to hide tooltip in resie